src/common/types/game_manager.ts

```diff
@@ -59,6 +59,7 @@ export interface GameManager {
   forceUninstall: (appName: string) => Promise<void>
   stop: (appName: string) => Promise<void>
   isGameAvailable: (appName: string) => boolean
+  pause: (appName: string) => Promise<void>
```

Comment:
Our store manager pattern should include functionality like this instead of adding if else statements in the Download Manager

I did not add `resume` as this would require refactoring significant state from download queue into the store managers and is outside the scope of this PR